spring-scaffold-cli/src/main/java/br/com/netodevel/command/template/TemplateHandler.java

```diff
@@ -3,10 +3,7 @@
 import br.com.generate.helpers.ScaffoldInfoHelper;
 import br.com.generator.core.GeneratorOptions;
 import br.com.templates_java.ComposeTemplate;
-import br.com.templates_java.config.jms_aws_sqs.EntryPointMessageGenerator;
-import br.com.templates_java.config.jms_aws_sqs.MessageListenerGenerator;
-import br.com.templates_java.config.jms_aws_sqs.SQSDependencyGenerator;
-import br.com.templates_java.config.jms_aws_sqs.SQSPropertiesGenerator;
+import br.com.templates_java.config.jms_aws_sqs.*;
 import joptsimple.OptionSet;
 import joptsimple.OptionSpec;
```

Comment:
The `ProducerMessageGenerator` is added but not removed `EntryPointMessageGenerator`, ensure the removal is intentional